core/src/main/java/org/jdbi/v3/Handle.java

```diff
@@ -28,109 +32,237 @@
 import org.jdbi.v3.tweak.ColumnMapper;
 import org.jdbi.v3.tweak.RowMapper;
 import org.jdbi.v3.tweak.StatementBuilder;
+import org.jdbi.v3.tweak.StatementCustomizer;
 import org.jdbi.v3.tweak.StatementLocator;
+import org.jdbi.v3.tweak.TransactionHandler;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 /**
  * This represents a connection to the database system. It usually is a wrapper around
  * a JDBC Connection object.
  */
-public interface Handle extends Closeable
+public class Handle implements Closeable
```

Comment:
I'm ambivalent, you can't subclass it anyway (no public or protected constructor).  So I'm not going to do it unless there's a compelling reason.